tests/apparmor-profiles/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Check apparmor profile generation for apps and hooks.
+prepare: |
+  for snap in basic-binaries basic-hooks
+  do
+    snapbuild ./../fixtures/snaps/$snap .
```

Comment:
I forgot to talk to @fgimenez about this. Can we please move these snaps to under tests/lib/snaps, so we have a single place for all test aiding content?

The "./" prefix may also be dropped here and on the rm below.